helix-rest/src/main/java/org/apache/helix/rest/server/ServerContext.java

```diff
@@ -278,6 +281,18 @@ public HelixDataAccessor getDataAccessor(String clusterName) {
     return _byteArrayZkBaseDataAccessor;
   }
 
+  public ZkBucketDataAccessor getZkBucketDataAccessor() {
+    if (_zkBucketDataAccessor == null) {
+      synchronized (this) {
+        if (_zkBucketDataAccessor == null) {
+          _zkBucketDataAccessor = new ZkBucketDataAccessor(_zkAddr);
```

Comment:
This still looks not right. Any client that is created without using createRealmAwareZkClient() is not compatible with the multi-ZK configuration, right?

Reply:
+1. I was trying to make this comment, for realm aware client, get a dedicated zk address client is not a good idea.

Reply:
ZkBucketDataAccessor will handle the multi-ZK configuration. This is a workaround since we can not create a zkClient from a zkClient.